pyproject.toml

```diff
@@ -78,6 +80,9 @@ addopts = """
 filterwarnings = "error"
 # Doctest python code in docs, python code in src docstrings, test functions in tests
 testpaths = "docs tests"
+env= [
+    "BEAMLINE=p46",
+]
```

Comment:
Must: The user is expected to do this themselves, this repo should not be coupled to a specific test rig. I would suggest just including something in either the readme or a comment in the system tests